lib/topological_inventory/ansible_tower/operations/core/topology_api_client.rb

```diff
@@ -18,8 +18,8 @@ def update_task(task_id, state:, status:, context:)
           end
 
           def svc_instance_url(service_instance)
-            rest_api_path = '/service_instances/{id}'.sub('{' + 'id' + '}', service_instance&.id.to_s)
```

Comment:
Before if the service_instance was nil this would make an API call to `/service_instances/` which doesn't mke a lot of sense